conjur/constants.py

```diff
@@ -36,3 +36,5 @@
 
 # For testing purposes
 TEST_HOSTNAME = "https://conjur-https"
+
+LOGIN_IS_REQUIRED = "To start using the CLI, log in to Conjur"
```

Comment:
When I run `conjur list` before init and login, I am prompted to initialize. At the end I get 
```
Successfully initialized the Conjur CLI
To start using the Conjur CLI, log in to the Conjur server by running `conjur login`
To start using the CLI, log in to Conjur
Enter your username:
```

Do you think this double "To start using" looks ok?

Reply:
`Successfully initialized the Conjur CLI.` remove period